### DISCUSSION TITLE
I think the "Getting Involved" links in the README need to be updated

### DISCUSSION THREAD

**Discussion by xayhewalo** [opds-community/drafts#61] [General]
Currently, the Slack link points to `http://readium-slack.herokuapp.com/` which says it's "offline for maintenance." Also, the latest notes from the monthly calls Google Drive link  `https://drive.google.com/drive/folders/1Qna-TwF9ZRNwCiu_3ZkEai6odUae-f1o`, dates back to 2018. 

Where should a newcomer to OPDS go to get involved/catch up on previous discussions?

**Comment by HadrienGardeur** [comment]
You're 100% right, we'll need to update this document.

We've been having calls on a monthly basis once again since the beginning of the year, here's a Google Doc with agenda and links to transcripts/recordings: https://docs.google.com/document/d/1759wJIS5pDVQz8fT18hk7tCV2JWMJAnR1sT5uWD9W6I/edit?usp=sharing

**Reply by xayhewalo** [reply]
@HadrienGardeur Do you have control over [www.opds.io](www.opds.io)? It similarly needs updated links. The site points to a google group that has not been used since 2019. I'd create an issue at [the website repo](https://github.com/opds-community/website), but it looks like that code is for a now defunct [discover-opds.org](discover-opds.org).

**Reply by HadrienGardeur** [reply]
Yeah, this is hosted on Github Pages from this same Github organization. I'll look into that as well.

**Comment by HadrienGardeur** [comment]
Done: https://github.com/opds-community/drafts/commit/061e138b443be91fcfef55d3f36d31075958af46
